lib-tests/bin/ci/test

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I like the start of some module loading tests. I want to iterate on how we structure them, though. We can talk about this on Monday and I can help lay out what I have in mind.